src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -125,18 +126,28 @@ export function getTransactionPayloadByInteractionType(
         TRANSACTION_DATA_KEYS[contractType][interactionType].functionName;
       switch (interactionType) {
         case ANT_INTERACTION_TYPES.SET_TICKER:
-        case ANT_INTERACTION_TYPES.SET_NAME: {
-          TRANSACTION_DATA_KEYS[contractType][interactionType].keys.forEach(
-            (key: string, index) => {
-              if (!txData[index]) {
-                throw new Error(
-                  `Missing key (${key}) from transaction data in the url. This may be due to the order of the data, the current order is [${txData}], the correct order is [${TRANSACTION_DATA_KEYS[contractType][interactionType].keys}]`,
-                );
-              }
-              payload[key] = txData[index];
-            },
-          );
-        }
+        case ANT_INTERACTION_TYPES.SET_NAME:
+          {
+            TRANSACTION_DATA_KEYS[contractType][interactionType].keys.forEach(
+              (key: string, index) => {
+                if (!txData[index]) {
+                  throw new Error(
+                    `Missing key (${key}) from transaction data in the url. This may be due to the order of the data, the current order is [${txData}], the correct order is [${TRANSACTION_DATA_KEYS[contractType][interactionType].keys}]`,
+                  );
+                }
+                payload[key] = txData[index];
+              },
+            );
+          }
+          break;
+        case ANT_INTERACTION_TYPES.SET_TARGET_ID:
+          {
+            payload[
+              TRANSACTION_DATA_KEYS[contractType][interactionType].keys[0]
```

Comment:
why do we need different logic here?

Reply:
good point, no reason not to do the same logic. Still needs the extra subdomain setting though. Thats there so we dont have to set it in jsx on the tables.

Reply:
added